packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -542,6 +544,7 @@ export default function ProjectNav({
             align="bottom"
             justify="end"
             usePortal={false}
+            darkMode={true}
```

Comment:
👍

Reply:
This is to keep the black background of the tooltip? I think I missed this naming, and I'm wondering if it makes sense.

If we have a global theme value for darkmode, when passed to tooltip I'd assume the background would be `white` so it would show for dark mode? If it was dark then how would it be visible?

I may be misunderstanding but figured I'd throw these thoughts out.

Reply:
Yeah, this is something I missed in the darkMode conversion. I don't know that darkMode as a prop makes sense for Tooltips. I actually could see us using a dark background / light text appearance for both dark and light mode to be honest, adjusting only for the contrast. That's more of a design decision though.

This change was just to fix a regression from that PR. Unless you're suggesting not including this change, probably best to talk about the impact of the prop change in the channel.

Reply:
Def fix the regression, was more a note on the conceptual design of a dark mode Tooltip.